cardano_node_tests/tests/test_governance.py

```diff
@@ -94,6 +97,10 @@ def test_create_and_answer_poll(
 
         # The poll should be signed with the delegateKey so that SPOS can verify it is a legit poll
         required_signer = cluster.g_genesis.genesis_keys.delegate_skeys[0]
+        required_signer_vkey = required_signer.parent / f"{required_signer.stem}.vkey"
+        required_signer_vkey_hash = cluster.g_genesis.get_genesis_vkey_hash(
+            vkey_file=required_signer_vkey
+        )
```

Comment:
It should be two different keys. Key can be specified either by skey file, or by vkey hash. If we use these two options with single key, it is not clear which one of these options was used in the end, and if the other one even works.